app/panel/actions/PanelActions.js

```diff
@@ -15,7 +15,8 @@ import {
 	TOGGLE_CLIQZ_FEATURE,
 	SHOW_NOTIFICATION,
 	CLOSE_NOTIFICATION,
-	TOGGLE_EXPERT
+	TOGGLE_EXPERT,
+	GET_SET_THEME
```

Comment:
@zarembsky could you rename these names to something a bit more clear please? (GET or SET?)